docker/Dockerfile.faux

```diff
@@ -45,6 +46,28 @@ COPY subset/bacnet/bacnetTests/src/main/resources/Faux*.json tmp/
 COPY --from=java_build /root/bacnet4j/bacnet4j-1.0-SNAPSHOT-all.jar bacnetTests/libs/
 RUN cd bacnetTests && ./gradlew build
 
+#Telnet dependancy
+RUN $AG update && $AG install -y telnetd
+RUN $AG update && $AG install xinetd
+RUN service xinetd start
```

Comment:
This runs the service during build time only, not at runtime... what is this accomplishing?

Reply:
moved to start_faux

Reply:
I don't understand... it's still here?  What did you move to start_faux?

Reply:
apologies I migrated the running of the service over to start_faux but accidentally left the line inside the Dockerfile